### DISCUSSION TITLE
Sengled E12-N1E Bulbs not reporting Power/Energy information

### DISCUSSION THREAD

**Discussion by jasonogren** [Koenkk/zigbee2mqtt#13927] [Devices]
I have multiple Sengled E12-N1E bulbs. Although they work correctly for operation under zigbee2mqtt, it appears that they don't include Power/Energy information. Previously I have connected these bulbs before to Home Assistant using ZHA and the bulbs provide two additional fields - Power (Wattage) and Energy (kWh). I have not seen this bulb in the release notes. 

It is the same issue as [(https://github.com/Koenkk/zigbee2mqtt/discussions/13302)] which has been resolved. I am unsure if I can use the same code and just change the bulb descriptors in external converter until it is added as well. Thanks.

**Comment by Koenkk** [comment]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
